src/Field.php

```diff
@@ -439,7 +439,7 @@ public function useAlias()
      */
     public function isEditable()
     {
-        return $this->read_only || $this->never_persist
+        return !$this->ui['editable'] && ($this->read_only || $this->never_persist)
```

Comment:
Probably this condition could be simplified to not check ui/editable multiple times:
```
return isset($this->ui['editable]) ? $this->ui['editable']
    : !( $this->read_only || $this->never_persist || $this->system );
```